scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala

```diff
@@ -64,4 +64,11 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
   it should "not provide .tupled in companion object with more than 22 fields" in {
     ArtisanalMoreThan22Fields.getClass.getMethods.map(_.getName) should not contain "tupled"
   }
+  
+  // This should compile
+  @BigQueryType.toTable
```

Comment:
I was wondering this and initially decided not to, because technically speaking we don't really care whether the type -say- has a .`tupled` method (at least for the scope of this issue). All we care about is whether it compiles, which for all intents and purposes is the same as not having `.tupled`, but it's not directly the thing we care about. Maybe we should though - am definitely not 100% on this.